contracts/src/intent/error.rs

```diff
@@ -5,24 +5,18 @@ use thiserror::Error as ThisError;
 pub enum IntentError {
     #[error("Borsh")]
     Borsh,
-    #[error("base64: {0}")]
+    #[error("Base64: {0}")]
```

Comment:
Hm, is there a convention for error messages in NEAR?
According to Rust guidelines, `Display` implementations for error messages [should](https://github.com/rust-lang/api-guidelines/blob/master/src/interoperability.md#error-types-are-meaningful-and-well-behaved-c-good-err) start with lowercase letter, so they can be then combined chained together in a more nice format